index.js

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable no-unused-vars */
```

Comment:
as suggested previously  Please do not disable linters. Linters enforce consistency and promote good coding practices. 🤝